### PR TITLE
like unlike post logic

### PR DESCRIPTION
**Pull request by divyayadav1606**

### CODE REVIEW

File: app/src/main/java/com/accountabilibuddies/accountabilibuddies/model/Post.java [hunk 1]
```diff
@@ -118,9 +118,20 @@ public void setCommentList(List<Comment> commentList) {
...
+
+    public void setLiked() {
+        this.liked = !liked;
+    }
+
     public List<ParseUser> getLikeList() {
+        List<ParseUser> users = (List<ParseUser>)get("likeList");
         //Set is_like here
-        return (List<ParseUser>) get("likeList");
+        if (users.contains(ParseUser.getCurrentUser()))
```
Comment (divyayadav1606) [inline app/src/main/java/com/accountabilibuddies/accountabilibuddies/model/Post.java:132]: this login does not work to search user in the Arraylist.

File: app/src/main/java/com/accountabilibuddies/accountabilibuddies/network/APIClient.java [hunk 2]
```diff
@@ -282,7 +282,7 @@ public void likeUnlikePost(String postId, boolean like, postListener listener) {
                 else
                     users.remove(ParseUser.getCurrentUser());
```
Comment (divyayadav1606) [inline app/src/main/java/com/accountabilibuddies/accountabilibuddies/network/APIClient.java:283]: Remove logic does not work here.

File: app/src/main/java/com/accountabilibuddies/accountabilibuddies/network/APIClient.java [hunk 3]
```diff
@@ -141,7 +141,7 @@ public void exitChallenge(String challengeObjectId, challengeListener listener)
             if (e == null) {
                 List<ParseUser> users = object.getUserList();
                 users.remove(ParseUser.getCurrentUser());
```
Comment (divyayadav1606) [inline app/src/main/java/com/accountabilibuddies/accountabilibuddies/network/APIClient.java:143]: Remove logic does not work here.

**Review by divyayadav1606 — COMMENTED** [review]
Please change add/Remove logic

**Review by deonna — APPROVED** [review]

**Pull request merged**